### PR TITLE
showcase code tab

### LINKED ISSUE #403: feature - create code tab for component showcase

### PR DESCRIPTION
**Pull request by odedre**
- forked `code-tab-tree` and created a new component - `teambit.code/panels/file-tree`. - the main diff is that it gets its tree nodes from outside so we can replace any problematic links.
- created `teambit.code/code`, which is the code tab in the showcase. does everything except for resolve of the scope url.
- added `teambit.code/code` to component showcase. 

closed #403

### CODE REVIEW

File: code/code/code.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  );
+
+  if (!code) return null;
+
+  return (
+    <SplitPane size="85%" layout={Layout.row} className={classNames(styles.code, styles[theme], className)} {...rest}>
+      <Pane>
+        <Editor
+          theme={theme === 'light' ? 'light' : 'vs-dark'}
+          height="90vh"
```
Comment (luvkapur) [inline code/code/code.tsx:78]: better to not use vh

File: code/code/tree-node/tree-node.module.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+.node {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-right: 8px;
+    > div:first-child {
```
Comment (luvkapur) [inline code/code/tree-node/tree-node.module.scss:7]: prefer to use specific classNames unless overriding external styles

File: code/panels/file-tree/file-tree.module.scss [hunk 3]
```diff
@@ -0,0 +1,28 @@
+.codeTabTree {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  width: 100%;
+  height: 100%;
+}
+
+.codeTabDrawer {
+  > div:first-child {
```
Comment (luvkapur) [inline code/panels/file-tree/file-tree.module.scss:10]: prefer to use specific css classes

File: code/panels/file-tree/file-tree.tsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+} & HTMLAttributes<HTMLDivElement>;
+
+export function FileTreePanel({
+  className,
+  fileTree,
+  dependencies,
+  currentFile = '',
+  FileTreeRenderer,
+}: FileTreePanelProps) {
+  const [openDrawerList, onToggleDrawer] = useState(['FILES' /* , 'DEPENDENCIES' */]);
```
Comment (luvkapur) [inline code/panels/file-tree/file-tree.tsx:27]: clean up

**Review by luvkapur — APPROVED** [review]

File: workspace.jsonc [hunk 5]
```diff
@@ -129,6 +134,7 @@
         "@teambit/explorer.ui.component-card": "0.0.9",
         "@teambit/explorer.ui.gallery.component-card": "^0.0.470",
         "@teambit/explorer.ui.gallery.component-grid": "^0.0.435",
+        "@teambit/graphql.contexts.graphql-provider": "0.0.7",
```
Comment (GiladShoham) [inline workspace.jsonc:137]: please use v0.0.8

**Review by GiladShoham — CHANGES_REQUESTED** [review]

**Pull request merged**